access/email/app.go

```diff
@@ -0,0 +1,483 @@
+package main
+
+import (
+	"context"
+	"time"
+
+	"github.com/gravitational/teleport-plugins/lib"
+	"github.com/gravitational/teleport-plugins/lib/logger"
+	"github.com/gravitational/teleport-plugins/lib/stringset"
+	"github.com/gravitational/teleport-plugins/lib/watcherjob"
+	"github.com/gravitational/teleport/api/client"
+	"github.com/gravitational/teleport/api/client/proto"
+	"github.com/gravitational/teleport/api/types"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/backoff"
```

Comment:
Once #295 or #212 is merged, we'll have our own `github.com/gravitational/teleport-plugins/lib/backoff` package. Lets prepare for this

```suggestion
	grpcbackoff "google.golang.org/grpc/backoff"
```

Reply:
Done